### DISCUSSION TITLE
Any one Knows about this

### DISCUSSION THREAD

**Discussion by yhak22** [skyline-emu/skyline#767] [Q&A] [answered]
hi guys 

i want to ask if any one know where can i find save games 
cause every time i download new ver , it erase my data and I can't find the folder for the emulator.

one thing ,
is there a plan to add save state for games

thank you skyline team :) 👍

**Comment by iniMerchant64** [comment]
Save data is in data/data/skyline.emu (needs root or adb to access). Save states are not only insanely hard but unpractical, their potenial size could go past 4gb

**Comment by PixelyIon** [comment, marked as answer]
Yes, we plan to make the save directory user-accessible and later when we're sign with the same keys there will be an option to save data during uninstallation so data can be retained across installs.
